src/site/_data/contributors.json

```diff
@@ -1272,5 +1272,22 @@
     "twitter": "rowan_m",
     "github": "rowan-m",
     "glitch": "rowan-m"
+  },
+  "addyosmani": {
```

Comment:
You should already be in this file: https://github.com/GoogleChrome/web.dev/blob/6b618d38d20d63ad431b9841c7de728a8f30a37d/src/site/_data/contributors.json#L817-L832

Reply:
Thanks! Updated commits.